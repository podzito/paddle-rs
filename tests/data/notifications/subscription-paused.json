{
  "data": {
    "id": "ntf_01j6fern2s2grg7mnqgrj5s79e",
    "type": "subscription.updated",
    "status": "not_attempted",
    "payload": {
      "data": {
        "id": "sub_01j6feh7txspm3a0y0eyhgrs23",
        "items": [
          {
            "price": {
              "id": "pri_01j52zfss0hcfxamnma1tf42gt",
              "name": "PodZito Pro",
              "type": "standard",
              "status": "active",
              "quantity": { "maximum": 1, "minimum": 1 },
              "tax_mode": "account_setting",
              "created_at": "2024-08-12T09:42:40.160401Z",
              "product_id": "pro_01j52z87e5sdt3p60mmtcavpgt",
              "unit_price": { "amount": "999", "currency_code": "USD" },
              "updated_at": "2024-08-29T11:35:06.922573Z",
              "custom_data": null,
              "description": "PodZito Pro - Monthly",
              "import_meta": null,
              "trial_period": { "interval": "day", "frequency": 7 },
              "billing_cycle": { "interval": "month", "frequency": 1 },
              "unit_price_overrides": []
            },
            "status": "active",
            "product": {
              "id": "pro_01j52z87e5sdt3p60mmtcavpgt",
              "name": "PodZito",
              "type": "standard",
              "status": "active",
              "image_url": null,
              "created_at": "2024-08-12T09:38:32.005Z",
              "updated_at": "2024-08-12T09:38:32.005Z",
              "custom_data": null,
              "description": "Podzito",
              "import_meta": null,
              "tax_category": "standard"
            },
            "quantity": 1,
            "recurring": true,
            "created_at": "2024-08-29T16:12:10.973Z",
            "updated_at": "2024-08-29T16:15:36.196Z",
            "trial_dates": { "ends_at": "2024-08-29T16:15:36.182Z", "starts_at": "2024-08-29T16:12:10.967Z" },
            "next_billed_at": null,
            "previously_billed_at": "2024-08-29T16:15:36.182Z"
          }
        ],
        "status": "paused",
        "discount": null,
        "paused_at": "2024-08-29T16:16:12.816Z",
        "address_id": "add_01j6fege3re5yv00dgrcfs1y6e",
        "created_at": "2024-08-29T16:12:10.973Z",
        "started_at": "2024-08-29T16:12:10.967Z",
        "updated_at": "2024-08-29T16:16:12.821Z",
        "business_id": null,
        "canceled_at": null,
        "custom_data": null,
        "customer_id": "ctm_01j535fawggst1gm3qsyanc0mb",
        "import_meta": null,
        "billing_cycle": { "interval": "month", "frequency": 1 },
        "currency_code": "USD",
        "next_billed_at": null,
        "billing_details": null,
        "collection_mode": "automatic",
        "first_billed_at": "2024-08-29T16:15:36.182Z",
        "scheduled_change": null,
        "current_billing_period": null
      },
      "event_id": "evt_01j6fermm868wv9g5qz06h2mg6",
      "event_type": "subscription.updated",
      "occurred_at": "2024-08-29T16:16:13.448587Z",
      "notification_id": "ntf_01j6fern2s2grg7mnqgrj5s79e"
    },
    "occurred_at": "2024-08-29T16:16:13.448587Z",
    "delivered_at": null,
    "replayed_at": null,
    "origin": "event",
    "last_attempt_at": null,
    "retry_at": null,
    "times_attempted": 0,
    "notification_setting_id": "ntfset_01j5akfpdszj2r2prnsjbf0ytt"
  },
  "meta": { "request_id": "7161d96f-3fe1-4e56-acd7-ce334e1c3478" }
}

