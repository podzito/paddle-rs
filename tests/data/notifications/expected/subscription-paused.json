{
  "event_id": "evt_01j6fermm868wv9g5qz06h2mg6",
  "notification_id": "ntf_01j6fern2s2grg7mnqgrj5s79e",
  "event": {
    "SubscriptionUpdated": {
      "billing_cycle": { "frequency": 1, "interval": "month" },
      "next_billed_at": null,
      "paused_at": "2024-08-29T16:16:12.816Z",
      "scheduled_change": null,
      "started_at": "2024-08-29T16:12:10.967Z",
      "items": [
        {
          "price": {
            "billing_cycle": { "frequency": 1, "interval": "month" },
            "created_at": "2024-08-12T09:42:40.160401Z",
            "custom_data": null,
            "description": "PodZito Pro - Monthly",
            "id": "pri_01j52zfss0hcfxamnma1tf42gt",
            "import_meta": null,
            "name": "PodZito Pro",
            "product_id": "pro_01j52z87e5sdt3p60mmtcavpgt",
            "quantity": { "maximum": 1, "minimum": 1 },
            "status": "active",
            "tax_mode": "account_setting",
            "trial_period": { "frequency": 7, "interval": "day" },
            "type": "standard",
            "unit_price": { "amount": "999", "currency_code": "USD" },
            "updated_at": "2024-08-29T11:35:06.922573Z"
          },
          "product": {
            "created_at": "2024-08-12T09:38:32.005Z",
            "custom_data": null,
            "description": "Podzito",
            "id": "pro_01j52z87e5sdt3p60mmtcavpgt",
            "image_url": null,
            "name": "PodZito",
            "status": "active",
            "tax_category": "standard",
            "type": "standard",
            "updated_at": "2024-08-12T09:38:32.005Z"
          },
          "created_at": "2024-08-29T16:12:10.973Z",
          "next_billed_at": null,
          "previously_billed_at": "2024-08-29T16:15:36.182Z",
          "quantity": 1,
          "recurring": true,
          "status": "active",
          "trial_dates": { "starts_at": "2024-08-29T16:12:10.967Z", "ends_at": "2024-08-29T16:15:36.182Z" },
          "updated_at": "2024-08-29T16:15:36.196Z"
        }
      ],
      "status": "paused",
      "transaction_id": null,
      "updated_at": "2024-08-29T16:16:12.821Z"
    }
  },
  "occurred_at": "2024-08-29T16:16:13.448587Z"
}
